src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  position: relative;
+  width: 100%;
+  margin-bottom: 20px;
+  display: block;
+}
+
+.icon {
+  position: absolute;
+  display: block;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon-SMALL {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  font-family: "Avenir", sans-serif;
```

Comment:
```suggestion
  font-family: inherit 
```
Specify font family under the `html` tag selector

Reply:
not fixed

move `font-family: "Avenir", sans-serif;` to `html` selector
and use `font-family: inherit` for input